CMakeLists.txt

```diff
@@ -1,7 +1,12 @@
-project(mallocMC)
-cmake_minimum_required(VERSION 2.8.12.2)
+project(mallocMC LANGUAGES CUDA CXX)
+cmake_minimum_required(VERSION 3.8)
+
+set(CMAKE_CXX_STANDARD 14)
```

Comment:
Do we use a feature which requires C++14 if not we should require C++11

Reply:
done